### PR TITLE
Reword about us slightly

### PR DESCRIPTION
**Pull request by canyon289**

### CODE REVIEW

**Review by OriolAbril — APPROVED** [review]

File: content/what-we-do/contents.lr [hunk 1]
```diff
@@ -123,8 +123,12 @@ If you meet one or more of these requirements, we are likely to add value to you
 * You make decisions with real-world consequences based on noisy data
     * In this case it is important to take all possible interpretations of your data into account, not just the most likely
     * Bayesian statistics naturally supports this by modeling beliefs as distributions rather than scalar values
-    * This results in more robust decisions that work across a whole range of plausible assumptions
+    * This results in more robust decisions that work across a whole range of plausible assumptions 
+* You or your team has an interest in bayesian statistics but doesn't know where to start
```
Comment (twiecki) [inline content/what-we-do/contents.lr]:
```suggestion
* You or your team has an interest in Bayesian statistics but doesn't know where to start
```

File: content/what-we-do/contents.lr [hunk 2]
```diff
@@ -123,8 +123,12 @@ If you meet one or more of these requirements, we are likely to add value to you
 * You make decisions with real-world consequences based on noisy data
     * In this case it is important to take all possible interpretations of your data into account, not just the most likely
     * Bayesian statistics naturally supports this by modeling beliefs as distributions rather than scalar values
-    * This results in more robust decisions that work across a whole range of plausible assumptions
+    * This results in more robust decisions that work across a whole range of plausible assumptions 
+* You or your team has an interest in Bayesian statistics but doesn't know where to start
```
Comment (twiecki) [inline content/what-we-do/contents.lr]:
```suggestion
* You or your team has an interest in Bayesian statistics but don't know where to start
```

Comment (twiecki) [inline content/what-we-do/contents.lr]:
```suggestion
* You or your team have an interest in Bayesian statistics but doesn't know where to start
```

**Review by twiecki — APPROVED** [review]

**Comment by twiecki** [comment]
Great stuff.

**Comment by canyon289** [comment]
Merging, we can do a follow up PR if theres more changes needed

**Pull request merged**